### ISSUE TITLE
Uncaught Less\Exception\Less_Exception_Parser: Expected ')' got ':'

### ISSUE THREAD

**Issue by ghost** [Asenar/less.php#5]
> Uncaught Less\Exception\Less_Exception_Parser: Expected ')' got ':' in image.less on line 7, column 16 05| 06| 07| .img-responsive(@display: block) { 08| display: @display; 09| max-width: 100%; 10| height: auto;

And that's the .less code (bootstrap.less 3.4.0) from bootstrap/mixins/images.less:
`.img-responsive(@display: block) {
    display: @display;
    max-width: 100%;
    height: auto;
}`

I had leafo's less compiler. I installed bootstrap.less version 3.4.0 and it crashed so I installed oyejorge's less compiler. Now I'm here cause oyejorge abandoned his project.
There is any less compiler (based on php) which is 100% stable?

Thanks in advance.

**Comment by Asenar** [comment]
Hi,

I also use bootstrap 3.4.0 and I don't reproduce your error, except if if I write something like this:

The error might happens where you call the function `.img-responsive(:something)`. somewhere.

**Comment by ghost** [comment]
> Hi,
> 
> I also use bootstrap 3.4.0 and I don't reproduce your error, except if if I write something like this:
> 
> The error might happens where you call the function `.img-responsive(:something)`. somewhere.

I don't call .img-responsive. Probably bootstrap itself does that.

**Comment by Asenar** [comment]
Yes, bootstrap call it natively, in `mixins/image.less` and `scaffolding.less` but the compilations works, so I suppose that's some part of your customization that uses `img-responsive()`.

As I said, I also use bootstrap 3.4.0 and I don't have this error and can't reproduce the error (I tried with less.php versions 1.8.0 and 1.7.0.14 and event 1.7.0.13).

Provide the code that provoke this error, or give the backtrace :)
